state/factory/factory.go

```diff
@@ -306,7 +295,7 @@ func (sf *factory) Height() (uint64, error) {
 func (sf *factory) NewWorkingSet() (WorkingSet, error) {
 	sf.mutex.RLock()
 	defer sf.mutex.RUnlock()
-	return NewWorkingSet(sf.currentChainHeight, sf.dao, sf.rootHash(), sf.actionHandlers)
+	return newWorkingSet(sf.currentChainHeight, sf.dao, sf.rootHash(), sf.actionHandlers, sf.saveHistory)
```

Comment:
cannot find newWorkingSet

Reply:
in workingset.go
just rename to lower-case since it does not need to be exported